nixio/dimensions.py

```diff
@@ -333,43 +333,68 @@ def _create_new(cls, parent, index, data_frame, column):
         """
         newdim = super(DataFrameDimension, cls)._create_new(parent, index)
         newdim.data_frame = data_frame
-        newdim.column = column
+        newdim.column_idx = column
         newdim._dimension_type = DimensionType.DataFrame
         return newdim
 
     def get_unit(self, index=None):
+        """
+        Get the unit of a specific column in the referenced DataFrame
```

Comment:
```suggestion
        Get the unit of the Dimension.  If an index is specified,
        it will return the unit of the column in the referenced DataFrame at that index.
```